gltfTutorial/README.md

```diff
@@ -0,0 +1,36 @@
+# glTF Tutorial
+
+This tutorial gives an introduction to [glTF](https://www.khronos.org/gltf), the GL transmission format. It summarizes the most important features and application cases of glTF, and describes the structure of the files that are related to glTF. It explains how glTF assets may be read, processed, and used to display 3D graphics efficiently.
+
+Some basic knowledge about [JSON](http://json.org/), the JavaScript Object Notation, is assumed. Additionally, a basic understanding of common graphics APIs, like OpenGL or WebGL, is required. The code examples will be given in [WebGL](https://www.khronos.org/webgl/).
+
+- [Introduction](gltfTutorial_001_Introduction.md)
+- [Basic glTF Structure](gltfTutorial_002_BasicGltfStructure.md)
+- [Example: A Minimal glTF File](gltfTutorial_003_MinimalGltfFile.md)
+- [Scenes and Nodes](gltfTutorial_004_ScenesNodes.md)
+- [Buffers, BufferViews, and Accessors](gltfTutorial_005_BuffersBufferViewsAccessors.md)
+- [Example: A Simple Animation](gltfTutorial_006_SimpleAnimation.md)
+- [Animations](gltfTutorial_007_Animations.md)
+- [Example: Simple Meshes](gltfTutorial_008_Meshes.md)
+- [Meshes](gltfTutorial_009_Meshes.md)
+- [Materials](gltfTutorial_010_Materials.md)
+- [Example: A Simple Material](gltfTutorial_011_SimpleMaterial.md)
+- [Programs and Shaders](gltfTutorial_012_ProgramsShaders.md)
+- [Materials and Techniques](gltfTutorial_013_MaterialsTechniques.md)
+- [Example: An Advanced Material](gltfTutorial_014_AdvancedMaterial.md)
+- (TBD: "Simple opacity": Explaining the `technique.states.functions` with the SimpleOpacity example)
+- [Example: A Simple Texture](gltfTutorial_015_SimpleTexture.md)
+- [Textures, Images, and Samplers](gltfTutorial_016_TexturesImagesSamplers.md)
+- [Example: Simple Cameras](gltfTutorial_017_SimpleCameras.md)
+- [Cameras](gltfTutorial_019_Cameras.md)
+- [Example: Simple Skin](gltfTutorial_019_SimpleSkin.md)
+- [Skins](gltfTutorial_020_Skins.md)
+- Extensions

```

Comment:
This as well as the TBD will likely not be part of the initial version. The extensions will be explained separately, and the Summary (locally) mainly contains code snippets for rendering with WebGL, which would go into a separate tutorial.